perceval/runtime/job_group.py

```diff
@@ -27,20 +27,30 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-from perceval.runtime import RemoteJob
+from perceval.runtime import RemoteJob, RunningStatus
 from perceval.runtime.rpc_handler import RPCHandler
 from perceval.utils import PersistentData, FileFormat
 from perceval.utils.logging import get_logger, channel
 import os
 import json
 from datetime import datetime
+from enum import Enum
+from tqdm import tqdm
+
 
 FILE_EXT_JGRP = 'jgrp'
 JGRP_DIR_NAME = "job_group"
 
 DATE_TIME_FORMAT = "%Y%m%d_%H%M%S"
 DEFAULT_MAX_SAMPLES = 10000
 
+class JobCategories(Enum):
```

Comment:
Should we do an Enum that "repeats" the information from the RunningStatus Enum?
I think using collections of Running status would make it more consistent

Reply:
Technically, this enum is for 'groups of runningstatus' and also includes handling of 'None' (this does not exist in RunningStatus and refers to jobs that were never sent to cloud). 

TBH, I would have preferred a better solution than mine. When you say "collections of RunningStatus" -> do you mean a list/dict/set/something else?